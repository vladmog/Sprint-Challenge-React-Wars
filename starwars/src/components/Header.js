import React from 'react';

function Header() {
    return (
        <div>
            <h1 className="Header">React Wars</h1>
        </div>
    )
}

export default Header;