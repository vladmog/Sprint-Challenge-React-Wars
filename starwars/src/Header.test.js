import React from 'react';
import renderer from 'react-test-renderer';

import Header from './components/Header';

describe('<Header />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});