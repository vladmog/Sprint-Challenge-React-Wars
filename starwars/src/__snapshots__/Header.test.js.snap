// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<div>
  <h1
    className="Header"
  >
    React Wars
  </h1>
</div>
`;
